charts/rd-judicial-api/values.yaml

```diff
@@ -13,7 +13,11 @@ java:
     OPEN_ID_API_BASE_URI: https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net/o
     LAUNCH_DARKLY_ENV: "{{ .Values.global.environment }}"
     ENVIRONMENT_NAME: "{{ .Values.global.environment }}"
-  image: 'hmctspublic.azurecr.io/rd/judicial-api:latest'
+    image: 'hmctspublic.azurecr.io/rd/judicial-api:latest'
```

Comment:
No need of adding image variable here. Do you have any explicit requirement to do add it here ?

Reply:
already existing only formatting done. Do we need to remove this yet ?